sonic_api.yaml

```diff
@@ -1335,8 +1335,6 @@ definitions:
         description: tunnel local termination IP address
   VnetEntry:
     type: object
-    required:
```

Comment:
This would require fix in https://github.com/Azure/sonic-restapi/blob/master/go-server-server/go/models.go#L321. However if vni is optional, is the expectation just to create a Vrf?